lib/main.js

```diff
@@ -25,6 +28,19 @@ let showSource;
 let disableExecuteTimeout;
 let excludedSniffs;
 
+const translateEncodingName = (name) => {
+  const canonical = iconvLiteAliases[name] || name;
+  let ret;
+  if (iconvTranslation[canonical]) {
+    ret = iconvTranslation[canonical];
```

Comment:
You could just return directly here, then there is no need for the temporary variable.